clj-refactor.el

```diff
@@ -221,6 +221,15 @@ won't run if there is a broken namespace in the project."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-ignore-libspecs
```

Comment:
hm.. I had not have a proper look but we might want to use this list to ignore stuff for other features as well perhaps?! if so this name is fine in terms of being generic enough